include/DDpackage.hpp

```diff
@@ -0,0 +1,2047 @@
+                      << "\n  CT Matrix Vector Multiplication size: " << sizeof(decltype(matrixVectorMultiplication)::Entry) << " bytes (aligned " << alignof(decltype(matrixVectorMultiplication)::Entry) << " bytes)"
+                      << "\n  CT Vector Inner Product size: " << sizeof(decltype(vectorInnerProduct)::Entry) << " bytes (aligned " << alignof(decltype(vectorInnerProduct)::Entry) << " bytes)"
+                      << "\n  CT Vector Kronecker size: " << sizeof(decltype(vectorKronecker)::Entry) << " bytes (aligned " << alignof(decltype(vectorKronecker)::Entry) << " bytes)"
+                      << "\n  CT Matrix Kronecker size: " << sizeof(decltype(matrixKronecker)::Entry) << " bytes (aligned " << alignof(decltype(matrixKronecker)::Entry) << " bytes)"
+                      << "\n  ToffoliTable::Entry size: " << sizeof(ToffoliTable<mEdge>::Entry) << " bytes (aligned " << alignof(ToffoliTable<mEdge>::Entry) << " bytes)"
+                      << "\n  Package size: " << sizeof(Package) << " bytes (aligned " << alignof(Package) << " bytes)"
+                      << "\n"
+                      << std::flush;
+        }
+
+        // print unique and compute table statistics
+        void statistics() {
+            std::cout << "DD statistics:" << std::endl
+                      << "[vUniqueTable] ";
+            vUniqueTable.printStatistics();
+            std::cout << "[mUniqueTable] ";
+            mUniqueTable.printStatistics();
+            std::cout << "[CT Vector Add] ";
+            vectorAdd.printStatistics();
+            std::cout << "[CT Matrix Add] ";
+            matrixAdd.printStatistics();
+            std::cout << "[CT Matrix Transpose] ";
+            matrixTranspose.printStatistics();
+            std::cout << "[CT Conjugate Matrix Transpose] ";
+            conjugateMatrixTranspose.printStatistics();
+            std::cout << "[CT Matrix Multiplication] ";
+            matrixMultiplication.printStatistics();
+            std::cout << "[CT Matrix Vector Multiplication] ";
+            matrixVectorMultiplication.printStatistics();
+            std::cout << "[CT Inner Product] ";
+            vectorInnerProduct.printStatistics();
+            std::cout << "[CT Vector Kronecker] ";
+            vectorKronecker.printStatistics();
+            std::cout << "[CT Matrix Kronecker] ";
+            matrixKronecker.printStatistics();
+            std::cout << "[Toffoli Table] ";
+            toffoliTable.printStatistics();
+            std::cout << "[Operation Table] ";
+            noiseOperationTable.printStatistics();
+            std::cout << "[ComplexTable] ";
+            cn.complexTable.printStatistics();
+        }
+    };
+
+    inline Package::vNode Package::vNode::terminalNode{.e   = {{{nullptr, Complex::zero}, {nullptr, Complex::zero}}},
```

Comment:
Designated initializers are C++20 ;)
(Below as well)

Reply:
I thought I got them all. Thanks for catching this.